python/ray/_private/node.py

```diff
@@ -206,8 +185,36 @@ def __init__(
                     f"{ray_params.dashboard_host}:{ray_params.dashboard_port}"
                 )
 
+        # It creates a session_dir.
         self._init_temp()
 
+        node_ip_address = ray_params.node_ip_address
+        if node_ip_address is None:
+            if connect_only:
+                node_ip_address = self._wait_for_node_address()
+            else:
+                node_ip_address = ray.util.get_node_ip_address()
+
+        assert node_ip_address is not None
+        ray_params.update_if_absent(
+            node_ip_address=node_ip_address, raylet_ip_address=node_ip_address
+        )
+        self._node_ip_address = node_ip_address
+        self._write_node_ip_address(node_ip_address)
+
+        if ray_params.raylet_ip_address:
```

Comment:
would you mind sharing again what's difference of `node_ip_address` vs `raylet_ip_address`?

Reply:
They are the same thing. I think we should remove `raylet_ip_address `